### PR TITLE
feat: update DAPI smoke tests

### PR DESCRIPTION
**Pull request by shumkov**
BREAKING CHANGE: <= 0.9 versions of DAPI won't be supported

### CODE REVIEW

File: test/smoke_web/dapi.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+      before(() => {
+        dapiClient = new DAPIClient();
+        dapiClient.getGrpcUrl = function getGrpcUrl() {
+          // eslint-disable-next-line no-underscore-dangle
+          return `${inventory._meta.hostvars[hostName].public_ip}:${variables.dapi_port}`;
+        };
+      });
+
+      it('should respond with a block for a selected height and hash and both should be equal', async () => {
```
Comment (shumkov) [inline test/smoke_web/dapi.js]: should respond via gRPC Web

File: karma.conf.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    ],
+    preprocessors: {
+      'lib/test/karma/loader.js': ['webpack'],
+    },
+    webpack: {
+      mode: 'development',
+      optimization: {
+        minimize: false,
+      },
+      plugins: [
```
[the remark this answers is not in the document]
Comment (jawid-h) [reply karma.conf.js]: That is our usual setup, so we can write using the same style as always.

[the remark this answers is not in the document]
Comment (shumkov) [reply karma.conf.js]: I thought we've added it only for rewiremock?

File: karma.conf.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        variables,
+      },
+    },
+  });
+  config.set({
+    frameworks: ['mocha', 'chai'],
+    files: [
+      'lib/test/karma/loader.js',
+    ],
+    exclude: [
```
Comment (shumkov) [inline karma.conf.js]: We can omit it right?

File: karma.conf.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (jawid-h) [reply karma.conf.js]: @shumkov Yep, removed.

File: karma.conf.js (hunk 3 above)
Comment (jawid-h) [reply karma.conf.js]: @shumkov Yep, removed.

**Comment by shumkov** [comment]
@jawid-h I think we need to make sure that it works, so let's deploy recent changes on Palinka first and then if tests are green we will merge it. Probably, some of the tests are wrong.

**Pull request merged**